src/pages/admin/AdminEvents.vue

```diff
@@ -7,22 +7,22 @@
     </div>
     <div class="events__body">
       <div class="events__search">
-        <TitledCard class="search__card" title="~~Event Search">
+        <TitledCard class="search__card" :title="$t('adminDashboard.event_search')">
```

Comment:
@aarontitus FYI, not that it matters too much, but purposely left out `$t` here cause the string input to that prop gets translated down the line. 

All good though, will just get "double translated" ;)